_episodes/03-terms.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Introduction"
```

Comment:
Should no longer be called 'Introduction', but something like 'important terms'?